### ISSUE TITLE
Accessibility: List Supported Accessibility Standards and Screen Readers and Browsers

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#1722]
**Is your feature request related to a problem? Please describe.**
We should list and document which browsers and standards we want to support.

**Describe the solution you'd like**

- [x]  list tested screen readers and versions docs 
- [x]  list supported guidelines we are going after - aka WCAG 2.1 
- [x]  list supported VPAT versions we recommend -  aka VPAT 2.2
- [x] Document this in README.MD or on a lint to ACCESSIBILITY.MD from the main readme.

**Decision Info on Screen Readers**
https://webaim.org/projects/screenreadersurvey7/#browsercombos is a bit old but provide some context. Initial thoughts but we want to limit this as much as possible.

Jaws - Chrome (I know IE 11 is old and Edge doesn’t work very well and Jaws 2019 is not perfect yet). Maybe 2018 with Firefox or Chrome? 
Voice Over - Mobile (IOS)
NVDA - Chrome or Firefox
Window Eyes - Was used by some of our government agencies so far since its free with office, but might be secondary to Jaws
Chrome Vox/Voice Over - Do these work well enough on Mac at this point for a mac screen reader?

**Comment by tmcconechy** [comment]
External Contractors gave some feedback on this issue and basically said it doesnt really matter too much which screen readers since they all work similar using the Accessibility DOM. So i think we can just go with a path of whats modernish and comfortable for use to test on and support.

Something mentioning that we test various screen readers and these are the main combinations we use, since all screen readers read the accessibility dom there may be minor differences in how things work but we should be mostly compatible so comes back to:

Jaws 2019 or 2018 -> Firefox or Chrome (For “compliant” option)
NVDA -> Firefox or Chrome (For “free” option)
Voice Over -> Safari (For Mac Option)
Voice Over -> Safari (For IOS Option)
Talkback -> Chrome (For Android Option)

**Comment by tmcconechy** [comment]
@nickwynja @clepore @EdwardCoyle -> I think we should just go with the above combination? Any feedback on this issue? (See previous reply)

**Comment by clepore** [comment]
If those are the most commonly used, then we should probably make sure we are compatible. I'm good with that.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
